bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -240,6 +240,8 @@ spec:
         resources:
 {{ toYaml .Values.metrics.resources | indent 10 }}
 {{- end }}
+{{ tpl (toYaml .Values.extraContainers) . | indent 6 }}
```

Comment:
```
{{- tpl (toYaml .Values.extraContainers) . | indent 6 }}
```
Adding the `-` if there are not `extraContainers` an empty space will not appear.

Reply:
I don't get this one :( I had to wrap it with `if` to apply only when `extraContainers` are not empty because it generated `[]` into statefulset.

Regarding whitespaces - cannot trim from left because I need newline. Example:
values.yaml:
```
extraContainers:
- name: container1
  image: image1
```
Output with `{{ tpl (toYaml .Values.extraContainers) . | indent 6 }}`:
```
        volumeMounts:                              
        - name: data        
          mountPath: /bitnami/mariadb
        - name: mariadb-galera-config                   
          mountPath: /opt/bitnami/mariadb/conf/my.cnf     
          subPath: my.cnf
      - image: image1         
        name: container1                             
```

Output with `{{- tpl (toYaml .Values.extraContainers) . | indent 6 }}`:
```
        volumeMounts:
        - name: data
          mountPath: /bitnami/mariadb
        - name: mariadb-galera-config
          mountPath: /opt/bitnami/mariadb/conf/my.cnf
          subPath: my.cnf      - image: image1
        name: container1
```

Reply:
Good.